code/Dependencies.md

```diff
@@ -38,3 +39,44 @@ To add a new dependency evaluate the following things:
 * Check how much it affects `storj/uplink` size.
 
 Note: many of these are fixable with communication or making our own PR-s.
+
+## Updating a dependency
```

Comment:
nit: it might be worth adding that reading a changelog is generally beneficial to get an overview of what the update consists of before jumping into the code.

Reply:
Ah, sorry, I missed it got merged just before I added an extra comment